### PR TITLE
chore: update policy identifiers

### PR DESCRIPTION
**Pull request by szenius**
This PR
* Updates policy identifiers to new shape
* Renames quota and transaction identifiers as identifierInputs for better differentiation
* Adds types to identifier inputs

(Test against [pr33](https://github.com/rationally-app/backend-api/pull/33))

### CODE REVIEW

File: src/services/policies/index.ts [hunk 1]
```diff
@@ -78,6 +78,24 @@ const mockGetPolicies = async (
...
+        order: 4,
+        quantity: { period: 1, limit: 1 },
+        identifiers: [
+          {
+            label: "Voucher code",
+            textInput: { visible: true, disabled: true, type: "STRING" },
+            scanButton: {
+              visible: true,
+              disabled: false,
+              type: "QR",
```
Comment (ilhaoni) [inline src/services/policies/index.ts:94]: actually, should we keep it as `type: "BARCODE"` haha

File: src/hooks/useCart/useCart.test.tsx [hunk 2]
```diff
@@ -38,8 +49,14 @@ const defaultProducts: Policy[] = [
       }
     },
     identifiers: [
-      { scannerType: "QR", label: "first" },
-      { scannerType: "QR", label: "last" }
+      {
+        ...defaultIdentifier,
+        label: "first"
```
Comment (ilhaoni) [inline src/hooks/useCart/useCart.test.tsx:54]: ehh, i'm a bit confused here, so there's 2 labels ah, one inside the defaultIdentifier, and one more outside? :O

File: src/services/policies/index.ts (hunk 1 above)
Comment (szenius) [reply src/services/policies/index.ts:94]: Oh right, can also~~ Not much impact now cuz it's not supported anyway HAHA

Comment (ilhaoni) [reply src/services/policies/index.ts:94]: HAHA, its okk~~ at least we make it accurate 😂  cause now we'll send the type into the db also haha

Comment (szenius) [reply src/services/policies/index.ts]: Ohhh, this one won't get sent into the db. It only gets activated when we run `npm run start:mock`, which mocks the backend~

**Pull request merged**